app/models/locations/location.rb

```diff
@@ -171,7 +171,7 @@ def remove_all_labwares(current_user)
     unknown_location = UnknownLocation.get
 
     # set the labwares to have location UnknownLocation rather than null
-    labwares.each do |labware|
+    labwares.find_each do |labware|
       labware.update(location: unknown_location, coordinate: nil)
```

Comment:
This last minute change actually broke some tests. See https://github.com/sanger/labwhere/pull/216/commits/b21faadf164723633b181ecc942da012e78beb01 for the nity-gritty details, and an explanation for why I chose to change the tests, rather than revert this change,